docs_src/use-cases/template.md

```diff
@@ -0,0 +1,78 @@
+## Use case template <!-- Replace with use case title -->
+This is a template for EdgeX use cases. The description of each section is included as comments and visible only when viewing the source. Remove this paragraph before submitting a use case.
+
+### Submitter(s)
+<!-- List use case submitters
+
+Format:
+- Name (Organization)
+-->
+
+### Status
+<!-- Status of the submission and date. 
+Status is one of: accepted, amended, deprecated
+Date is ISO 8601 (YYYY-MM-DD)
+
+E.g.:
+- [accepted](URL of PR) (2022-04-01)
+- [amended](URL of PR) (2022-05-01)
+-->
+
+### Category
+<!-- What IoT market segments does this use case address? 
+Examples are:
+- Home (home automation, home improvement, energy efficiency)
+- Lifestyle (wearable computing, entertainment and music, family, leisure, pets, toys, drones)
+- Health (fitness, monitoring, measurement, diagnosis, surgery, patience care)
+- Mobility (connected cars, ebikes, aerospace & airports, marine, rail & stations, automotive, traffic)
+- Retail (stores, shops, convenience)
+- Energy (transmission & distribution, fossil, nuclear, alternative)
+- Cities (infrastructure, water & wastewater, HVAC, lighting, security, life safety)
+- Manufacturing (mining, oil & gas, production, supply chain)
+- Public & Services (schools, universities, government, banking, insurance, administration, commercial services)
+- Other (environment, military, agriculture, hospitality)
+
+Reference and more details: https://iot-analytics.com/iot-market-segments-analysis/
+-->
+
+### Motivation
+<!-- Summarize the problems that are solved by the use case and why they important in the relevant domain -->
+
+### Target Users
+<!-- List all stakeholders that are involved in the use case from the following list:
+- Device Manufacturer
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud Provider
+- Service Provider
+- Network Operator
+- Software Developer
+- Software Deployer
+-->
+
+### Description
+<!-- Provide the description of the use case from target users' perspective -->
+
+### Existing solutions
+<!-- How is the given use case currently implemented in the industry? List and describe each approach. Highlight possible gaps. -->
+
+### Requirements
+<!-- Provide a list of (non-)functional requirements that aren't addressed by EdgeX with references to corresponding feature request Github issues.
+Include security and privacy requirements that are specific to the given domain.
+
+Format:
+- Summary ([repo#issue-num](URL))
+-->
```

Comment:
I thought about throwing in a fully-fledged template for the requirements, such as the [Volere template](https://www.reqview.com/blog/2019-02-27-news-volere-requirements-specification-template/) but then realized that it is hard to reference each requirement or to identify duplicates across use cases. Moreover, they directly overlap with feature requests which do have a template asking for the following questions:
- Can you pin-point one or more EdgeX services the are relevant for this feature request?
- A clear and concise description of the problem or missing capability...
- If you have a solution in mind, please describe it.
- Have you considered any alternative solutions or workarounds?

A question that is missing from the feature request template is who is the target user? Who needs the feature?

I think the proposed template is simple and sufficient by giving a short requirement summary and referencing the feature request that includes more technical detail. Multiple use cases can reference the same feature request. ADRs can reference the same. PRs can close those, and the use case will eventually have references to Done issues with no editorial effort.

Reply:
Agree with all of the above Farshid.  Keep it simple to start.  If we are finding we need additional details/information, we can ask the requestor and start to add to future templates.